worker/src/main/java/gov/cms/ab2d/worker/processor/CoverageMappingCallable.java

```diff
@@ -0,0 +1,296 @@
+package gov.cms.ab2d.worker.processor;
+
+import ca.uhn.fhir.rest.server.exceptions.InternalErrorException;
+import gov.cms.ab2d.bfd.client.BFDClient;
+import gov.cms.ab2d.common.model.CoverageMapping;
+import gov.cms.ab2d.common.model.Identifiers;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.hl7.fhir.dstu3.model.*;
+
+import java.util.*;
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.*;
+
+@Slf4j
+public class CoverageMappingCallable implements Callable<CoverageMapping> {
+
+    static final String BENEFICIARY_ID = "https://bluebutton.cms.gov/resources/variables/bene_id";
+    public static final String CURRENCY_IDENTIFIER =
+            "https://bluebutton.cms.gov/resources/codesystem/identifier-currency";
+    public static final String CURRENT_MBI = "current";
+    public static final String HISTORIC_MBI = "historic";
+    public static final String MBI_ID = "http://hl7.org/fhir/sid/us-mbi";
```

Comment:
These are duplicate with those in PatientContraceCallable.  Just move them to something like a MbiSupport class.

Reply:
The PatientContractCallable will be deleted in the future and replaced by this class because we are loading the data into our database beforehand. Not sure if we should make a class when the intention is for the PatientContractCallable to be deleted in the near future.